kata-agent.service.in

```diff
@@ -10,5 +16,4 @@ StandardOutput=tty
 Type=simple
 ExecStart=@bindir@/@kata-agent@
 LimitNOFILE=infinity
-ExecStop=/bin/sync ; /usr/bin/systemctl --force poweroff
-FailureAction=poweroff
+ExecStop=/bin/sync
```

Comment:
do we even need the sync? I guess maybe we are trying to flush fs buffers out of the VM back to the host?

Reply:
@grahamwhaley Yes, we need the sync for block based storage drivers to flush the buffers to the block device.